blog/2021-05-29-using-rust.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Building a reliable and tRUSTworthy web service 🦀
+author: Rohan
+github: RohanGautam
+description: Explore the use of rust in FoodDX, and go over key architectural decisions
+tags: rust
+---
+
+## Product intro
+
+[FoodDX](https://www.fooddx.com/) is a service which helps people get insights on how to improve their diet with the help from our proprietary AI technology and personalized feedback from our nutrition experts. In its current stage, it helps score images of food taken through an app, and gives it a score from 1-5, and also provides personalized tips for the food.
+
+It's a big project with a lot of components - the AI models, the app, and the backend infrastructure which handles it all. We'll be taking a look at the backend in this article.
+
+## Lifecycle of an image
+
+Before we dive into the infrastructure, it would help to take a look at what an image goes through once it enters our system.
+
+- The image is assigned a UUID.
+- The image is uploaded to a `s3` bucket.
+- The image is then downloaded, preprocessed, run through our models, and a food tip is generated for the image.
+- The food tip, inference results and the image hash [^1] form the final response.
```

Comment:
Is there a way to link these to the bottom explanations/elaborations of the footer comments? Not a big deal tho

Reply:
yeah! the [^1] shows up as a link to the comments

Reply:
![image](https://user-images.githubusercontent.com/17317792/119948843-d81d5780-bfcb-11eb-8c7e-5b67b5b72ec8.png)
like dis